source/_integrations/iss.markdown

```diff
@@ -18,33 +20,13 @@ This means that ISS is 10° above the horizon of your home.
 You can check in the attributes of the sensor to see the timestamp for the next
 rise of the station, its current coordinates, and the number of people in space.
 
-## Configuration
-
-To add ISS binary sensor to your installation,
-add the following to your `configuration.yaml` file:
-
-```yaml
-# Example configuration.yaml entry
-binary_sensor:
-  - platform: iss
-```
-
-{% configuration %}
-name:
-  description: The name for this sensor in the frontend.
-  required: false
-  type: string
-  default: ISS
-show_on_map:
-  description: Option to show the position of the ISS on the map.
-  required: false
-  type: boolean
-  default: false
-{% endconfiguration %}
+{% include integrations/config_flow.md %}
 
 <div class='note warning'>
 
-If you set `show_on_map: true` then the location attributes are named `latitude` and `longitude`.
-The default name of the location attributes is `lat` and `long` to avoid showing them on the map.
+If you enable "Show on map" when configurating this integration then the location attributes 
```

Comment:
This should not be part of the configuration, but part of an option flow, which needs to be described differently.

Reply:
```suggestion
If you enable "Show on map" in the options for this integration then the location attributes 
```